site/docs/pages/wallet/wallet.mdx

```diff
@@ -84,7 +81,6 @@ export function WalletComponents() {
       <Identity className="px-4 pt-3 pb-2" hasCopyAddressOnClick>
         <Avatar />
         <Name>
-          <Badge />
```

Comment:
Something is missing here? :)
I let you guess what's missing

Reply:
Hmm do we want it displayed as `<Name />`

Reply:
yup :)

Reply:
haha sweet, that's the only thing I could think of 😆